src/core_codemods/tempfile_mktemp.py

```diff
@@ -19,25 +23,64 @@ class TempfileMktempTransformer(
     change_description = "Replaces `tempfile.mktemp` with `tempfile.mkstemp`."
     _module_name = "tempfile"
 
-    def leave_SimpleStatementLine(self, original_node, updated_node):
+    def __init__(
+        self,
+        context: CodemodContext,
+        results: list[Result] | None,
+        file_context: FileContext,
+        _transformer: bool = False,
+    ):
+        self.mktemp_calls: set[cst.Call] = set()
+        super().__init__(context, results, file_context, _transformer)
+
+    def visit_Call(self, node: cst.Call) -> None:
+        self.mktemp_calls.add(node) if self._is_mktemp_call(node) else None
```

Comment:
The `else` is useless here.

Reply:
you can't write a ternary expression in Python without an else. Just wanted to make it one line instead of
```
if self._is_mktemp_call(node):
    self.mktemp_calls.add(node)
```